wremnants/__init__.py

```diff
@@ -4,12 +4,12 @@
 
 ROOT.gInterpreter.AddIncludePath(f"{pathlib.Path(__file__).parent}/include/")
 
-ROOT.gInterpreter.Declare('#include "muonCorr.h"')
-ROOT.gInterpreter.Declare('#include "histoScaling.h"')
-ROOT.gInterpreter.Declare('#include "histHelpers.h"')
-ROOT.gInterpreter.Declare('#include "utils.h"')
-ROOT.gInterpreter.Declare('#include "csVariables.h"')
-ROOT.gInterpreter.Declare('#include "EtaPtCorrelatedEfficiency.h"')
+narf.clingutils.Declare('#include "muonCorr.h"')
```

Comment:
Should we take this as the time to move all this stuff out of here?

Reply:
Let's do this in a subsequent PR.  I want to do this 1:1 as part of hardening the code and making sure there are no errors in conjunction with the singularity/root update with better error handling for the templates.

Reply:
alrighty